src/pivotal-ui/components/meta/meta.scss

```diff
@@ -134,6 +134,78 @@ parent: meta
 ```
 */
 
+/*doc
+---
+title: Need Help Orgs Module
+name: 02_meta_need_help_orgs
+parent: meta
+---
+
+```html_example
+<svg style="display: none;">
+  <symbol id="i-498-twitter" viewBox="0 0 28.002 22.756">
+      <path d="M28.002 2.694c-1.03.457-2.138.766-3.3.904A5.752 5.752 0 0 0 27.228.42a11.49 11.49 0 0 1-3.648 1.394 5.744 5.744 0 0 0-9.936 3.93c0 .45.05.89.148 1.31a16.318 16.318 0 0 1-11.84-6 5.716 5.716 0 0 0-.778 2.886c0 1.993.95 3.75 2.493 4.78-.94-.03-1.665-.287-2.665-.718v.072c0 2.783 2.043 5.105 4.67 5.634a5.49 5.49 0 0 1-2.547.098 5.762 5.762 0 0 0 5.374 3.99 11.517 11.517 0 0 1-7.13 2.46c-.467 0-.92-.028-1.37-.082a16.26 16.26 0 0 0 8.807 2.58c10.565 0 16.343-8.752 16.343-16.344 0-.25-.003-.497-.014-.743a11.642 11.642 0 0 0 2.866-2.973z"/>
+  </symbol>
+
+  <symbol id="i-558-mailbox" viewBox="0 0 24 28">
+    <path d="M22 8H11V6h7l-2-2.913L18 0H9v8H2a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h10v4h4v-4h6a2 2 0 0 0 2-2V10a2 2 0 0 0-2-2zm0 14H4V10h5v4.277c-.596.347-1 .985-1 1.723a2 2 0 0 0 4 0c0-.738-.404-1.376-1-1.723V10h11v12z"/>
+  </symbol>
+</svg>
+
+<div class="row">
+  <div class="col-md-8">
+    <div class="panel panel-basic-alt bg-neutral-10 pvxxl phxl" id="need-help-expandable-orgs">
+
+      <div class="row">
+        <div class="col-xs-7 col-xs-offset-1 txt-c">
+          <img alt="support avatar" class="avatar" height="60" src="styleguide/avatar.svg" width="60">
+          <!-- {{svg "@npmcorp/pui-css-meta/orgs-svgeople-avatar.svg" alt="support avatar" class="avatar" height="60" width="60"}} -->
```

Comment:
Yup. I did in one I did ... I really want to get hbs working in the styleguide for this.